tests/e2e/dsc_creation_test.go

```diff
@@ -424,6 +429,42 @@ func (tc *testContext) testDefaultCertsAvailable() error {
 	return nil
 }
 
+func (tc *testContext) testDefaultModelRegistryCertAvailable() error {
```

Comment:
this is very ModelReg speicific test case, do we want to add it here? 
it is supposed to be general functions regardless which component

Reply:
Is there a better place for the cert creation verification. I noticed the above `testDefaultCertsAvailable()` test for kserver, so I figured this was the place to do the same for model registry.